### PR TITLE
fix: remove url dependency in the browser

### PR DESCRIPTION
**Pull request by hugomrdias**
This PR reduces bundle size (OKR work).

26.86KB > 22.63KB -15.74%

Basically removes url dep in the browser bundle. 

Needs
 - [x] https://github.com/ipfs/aegir/pull/307

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,73 +1,74 @@
...
-    "Joe Hildebrand <joe+github@cursive.net>",
-    "Patrick Gansterer <paroga@paroga.com>",
-    "Ryan Pendergast <ryan.pendergast@gmail.com>",
-    "Rüdiger Klaehn <rklaehn@gmail.com>",
-    "Scorpi <cobblebone@gmail.com>",
-    "Tobias Wich <tobias.wich@ecsec.de>",
-    "dignifiedquire <dignifiedquire@gmail.com>",
-    "greenkeeperio-bot <support@greenkeeper.io>"
-  ]
+    "name": "borc",
```
Comment (dignifiedquire) [inline package.json]: can you please leave the indent as it was?

File: .travis.yml [hunk 2]
```diff
@@ -3,8 +3,6 @@ language: node_js
 
 matrix:
   include:
-  - node_js: 8
```
Comment (dignifiedquire) [inline .travis.yml]: why is node 8 not supported anymore?

**Comment by hugomrdias** [comment]
should be fixed now

**Comment by hugomrdias** [comment]
this PR is ready to merge

**Review by vmx — COMMENTED** [review]
Looks good to me, but I'd like to get an approval from @dignifiedquire.

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,18397 @@
+{
```
Comment (dignifiedquire) [inline package-lock.json]: can we please not add this? if this is important, please put it into its own PR with a description why.

File: benchmarks/index.js [hunk 4]
```diff
@@ -40,9 +40,9 @@ suite.add(`encode - borc - ${vecLength}`, () => {
 })
 
 suite.add(`encode - stream - borc - ${vecLength}`, () => {
-  const enc = new fastCbor.Encoder({stream (chunk) {
+  const enc = new fastCbor.Encoder({ stream (chunk) {
```
Comment (dignifiedquire) [inline benchmarks/index.js:43]: please remove all the white space changes, unless they have to be there to make linting pass

File: .travis.yml [hunk 5]
```diff
@@ -3,7 +3,7 @@ language: node_js
...
+  - node_js: 8
```
Comment (dignifiedquire) [inline .travis.yml]: superfluous whitespace

**Review by dignifiedquire — CHANGES_REQUESTED** [review]
- actual content changes lgtm
- please drop the unrelated changes (package-lock & white space diff) 
- nitpick: this is not really a `fix`, but rather a `feat`, or `refactor`

**Comment by hugomrdias** [comment]
fixed all feedback and squashed
white space is needed because aegir uses a new config

**Review by dignifiedquire — APPROVED** [review]
thanks

**Comment by hugomrdias** [comment]
can you release pls ? so this starts bubbling up

**Pull request merged**